### PR TITLE
Updated CONTRIBUTING.md 

### PR DESCRIPTION
**Pull request by debo19**
Signed-off-by: Debopriya Bhattacharjee <debopriyabhattacharjee@Debopriyas-MacBook-Air.local>

**Description**
Updated CONTRIBUTING.md with new practices with respect to images and icons.

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 8f5f8d9961b1eedb82fc499825d2caa32513063d at: https://61f1667e4efc61c8e54b4c0d--layer5ng.netlify.app

**Comment by leecalcote** [comment]
Oh, sure. Rub it in. "Debopriyas-MacBook-Air.local"

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -51,6 +51,18 @@ If you'd like to contribute a post to the layer5.io/blog, please open an issue a
 
 The following list of instructions pertain to commonplace site updates by contributors.
 
+## Things to keep in mind
+
+  1. Use only the theme colors available at [themeStyles.js](https://github.com/layer5io/layer5/blob/master/src/theme/app/themeStyles.js) instead of external color codes.
+  2. Avoid adding `rel="noreferrer"` attribute to internal links on the site.
```
Comment (leecalcote) [inline CONTRIBUTING.md:57]: Reinforced by an eslint pre-commit hook, too.

**Review by leecalcote — APPROVED** [review]

**Review by Chadha93 — APPROVED** [review]
Looks great :+1:

File: CONTRIBUTING.md (hunk 1 above)
Comment (Nikhil-Ladha) [reply CONTRIBUTING.md:57]: You mean this needs to be configured, right?
Or, we are good with this PR and configure it later?

**Pull request merged**